machina/apps/forum_member/utils.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.mail import EmailMultiAlternatives
+from django.template import loader
+
+
+def send_notification(user, post):
+    """
+    Send notification on email to the user for the given post.
+    """
+    if not user.email:
+        return
+
+    context = {
+        'user': user,
+        'post': post,
+        'topic': post.topic,
+        'current_site': Site.objects.get_current(),
+    }
+    template = 'forum_member/emails/notification'
+
+    # Load templates for subject and text content and render with context
+    subject = loader.get_template(
+        '{}.subject.txt'.format(template)).render(context)
+    text_content = loader.get_template(
+        '{}.txt'.format(template)).render(context)
+
+    msg = EmailMultiAlternatives(
+        # The subject might contain newlines etc. so strip and combine it
+        # into one line.
+        ' '.join(subject.splitlines()).strip(),
+        text_content,
```

Comment:
Yeah I'll create a HTML version and a simple `BaseEmail` class that can easily be overridden.